src/com/sun/ts/tests/signaturetest/jsonp/record.sh

```diff
@@ -21,7 +21,7 @@
 
 PATHSEP=:
 PWD=`pwd`
-JSONP_CLASSES=/files/jsonp/ri/lib/javax.json-1.0-SNAPSHOT.jar
+JSONP_CLASSES=/files/jsonp/ri/lib/jakarta.json-1.0-SNAPSHOT.jar
```

Comment:
@alwin-joseph should ^ reference jakarta.json.jar instead of jakarta.json-1.0-SNAPSHOT.jar?

Reply:
jakarta.json-api.jar would be more generic and apt here.